domains/dior.json

```diff
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "ewcstasy",
+        "email": "steriods@internet.ru"
+    },
+    "record": {
+        "CNAME": "ewcstasy.github.io"
```

Comment:
Both preview and site have little content provided on site. Reopen this PR when you have completed your site.